harvester/extract/waf.py

```diff
@@ -0,0 +1,29 @@
+import requests
+from bs4 import BeautifulSoup
+import os
+
+
+def traverse_waf(url, files=[], file_ext=".xml", folder="/", filters=[]):
+    parent = os.path.dirname(url.rstrip("/"))
+
+    res = requests.get(url)
+    if res.status_code == 200:
```

Comment:
can you add some pseudo code, or a TODO, for error handling?

Reply:
[added](https://github.com/GSA/datagov-harvesting-logic/pull/24/commits/aee51a627bc2ce42387c8131996c14b0ccdb82e9)